features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -97,20 +98,35 @@ const ActionConfigurationPage: FunctionComponent<ActionConfigurationPageInterfac
         mutate: mutateActions
     } = useGetActionsByType(actionTypeApiPath);
 
-    const isLoading: boolean = isActionsLoading || !actions || !Array.isArray(actions);
+    const actionId: string = useMemo(() => {
+        if (actions && actions.length >= 1) {
+            return actions[0].id;
+        } else {
+            return null;
+        }
+    }, [ actions ]);
```

Comment:
More simplified version;
```
useMemo(() => actions?.[0]?.id || null, [actions]);
```
You can utilize null checks to reduce if-else conditions.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7168/commits/1ba4f62f4c5620a326ae4ee9e34ab74d23d98e0b